src/Paket.PowerShell/PowerShell.fs

```diff
@@ -25,7 +25,10 @@ module PaketPs =
         async {
             try
                 do! Async.SwitchToNewThread()
-                do! computation
+                try
```

Comment:
Without the `try with` here, an exception kills the started thread, which kills the powershell thread which kills powershell. It is important.